syntax = "proto3";

package demo;

message Response {
    string server_id = 1;
    int32 response_data = 2;
}

message FileChunk {
    string client_id = 1;
    string file_name = 2;
    bytes chunk = 3;
}

service GRPCDemo {
    rpc ClientStreamingMethod (stream FileChunk) returns (Response);
}

